scene/3d/navigation_agent_3d.cpp

```diff
@@ -111,8 +111,8 @@ void NavigationAgent3D::_bind_methods() {
 	ADD_PROPERTY(PropertyInfo(Variant::FLOAT, "radius", PROPERTY_HINT_RANGE, "0.1,100,0.01,suffix:m"), "set_radius", "get_radius");
 	ADD_PROPERTY(PropertyInfo(Variant::FLOAT, "neighbor_distance", PROPERTY_HINT_RANGE, "0.1,10000,0.01,suffix:m"), "set_neighbor_distance", "get_neighbor_distance");
 	ADD_PROPERTY(PropertyInfo(Variant::INT, "max_neighbors", PROPERTY_HINT_RANGE, "1,10000,1"), "set_max_neighbors", "get_max_neighbors");
-	ADD_PROPERTY(PropertyInfo(Variant::FLOAT, "time_horizon", PROPERTY_HINT_RANGE, "0.01,100,0.01,suffix:s"), "set_time_horizon", "get_time_horizon");
-	ADD_PROPERTY(PropertyInfo(Variant::FLOAT, "max_speed", PROPERTY_HINT_RANGE, "0.1,10000,0.01,suffix:m/s"), "set_max_speed", "get_max_speed");
+	ADD_PROPERTY(PropertyInfo(Variant::FLOAT, "time_horizon", PROPERTY_HINT_RANGE, "0.01,10,0.01,suffix:s"), "set_time_horizon", "get_time_horizon");
+	ADD_PROPERTY(PropertyInfo(Variant::FLOAT, "max_speed", PROPERTY_HINT_RANGE, "0.1,1000,0.01,suffix:m/s"), "set_max_speed", "get_max_speed");
```

Comment:
Are higher values really never going to be needed? Or should it use `,or_greater` to allow going beyond the max if wanted with manual input (but the slider stays in the given range)?

Reply:
In that case we should add 'or_greater' to every property of both NavigationAgent2D and 3D cause there is no real "technical" limit to these properties, just a practical one.

The current limits have some merit cause everything beyond that values turns the avoidance more or less ridiculous as radius gets sqrt and multiplied by speed and time horizon. Might as well ask for a crystal ball to predict the future at that point instead of a useful avoidance velocity.